ui/app/styles/components/toggle.scss

```diff
@@ -0,0 +1,75 @@
+$size: 10px;
```

Comment:
This should be 12px so that there is a 2px padding on the top, bottom, and outer edge of the 8px knob.

Reply:
Tweaked some numbers:
![image](https://user-images.githubusercontent.com/174740/70765317-4191a380-1d0f-11ea-8c95-fdde43d07dcc.png)